.circleci/config.yml

```diff
@@ -104,3 +115,4 @@ workflows:
           context: profile-databricks
           requires:
             - integration-postgres
+      - integration-trino
```

Comment:
Could you please just add a 
```
          requires:
            - integration-postgres
```
like the other Cloud Warehouses? The idea is that we try first with the "offline" ones so we don't incur any Cloud cost if those fail.

Reply:
Done!
I've also added `context: profile-trino`